clean up docs todos (#6886)
RasaHQ/rasa: docs/docs/domain.mdx

title: Domain
---

The `Domain` defines the universe in which your assistant operates.
It specifies the `intents`, `entities`, `slots`, `responses`, `forms`, and `actions`
your bot should know about. It also defines a `session_config` to configure
conversation sessions.

> As an example, here the domain of the 

Reviewer: ```suggestion
As an example, here is the domain of the 
```

This means that excluded entities for those intents will be unfeaturized and therefore
will not impact the next action predictions. This is useful when you have
an intent where you don't care about the entities being picked up. 

> If you list your intents as normal without this parameter, the entities will be

Reviewer: ```suggestion
If you list your intents without this parameter, the entities will be
```

[Customizing the session start action](./default-actions.mdx#customization) shows you how to do that.

:::

## Config 

> The `config` key in the domain file keeps track of one parameter, `store_entities_as_slots`.

Reviewer: ```suggestion
The `config` key in the domain file maintains the `store_entities_as_slots` parameter. 
```

> When an entity is recognized by the NLU model, if the entity type matches a slot name,

Reviewer: ```suggestion
When an entity is recognized by the NLU model and the entity name matches a slot name,
```

Author: So much clearer thank you 🙌

> this value defines whether the entity value should be placed in that slot.

Reviewer: ```suggestion
`store_entities_as_slots` defines whether the entity value should be placed in that slot.
```

By default, entities will auto-fill slots of the same name. 

> You can turn off all slot auto-filling by changing the `store_entities_as_slots` parameter: 

Reviewer: ```suggestion
You can turn off all slot auto-filling by setting the `store_entities_as_slots` parameter to `false`: 
```

```yaml-rasa title="domain.yml"
config:
  store_entities_as_slots: false
```

> You can also turn off this behavior for a specific slot by using the [`auto_fill`](./domain.mdx#slot-auto-fill)

Reviewer: ```suggestion
You can also turn off this behavior for a specific slot using the [`auto_fill`](./domain.mdx#slot-auto-fill)
```
